CMakeLists.txt

```diff
@@ -69,7 +69,12 @@ set(IGN_MSGS_VER ${ignition-msgs5_VERSION_MAJOR})
 
 #--------------------------------------
 # Find if ign command is available
-find_program(HAVE_IGN_TOOLS ign)
+
+ign_find_package(ignition-tools QUIET)
```

Comment:
This is reverting https://github.com/gazebosim/gz-gui/pull/240. My understanding is that the CMake / pkg-config module is being removed from `ign-tools` (see https://github.com/gazebo-tooling/release-tools/issues/472), so this won't work anymore.

Reply:
Reverted in [fe2be11](https://github.com/gazebosim/gz-gui/pull/392/commits/fe2be11e57bf2c3cc2b838f9d5144167a43e3aef) and added a note to avoid future confusion.